apps/andi/lib/andi_web/live/ingestion_live_view/finalize/finalize_form.ex

```diff
@@ -60,7 +60,7 @@ defmodule AndiWeb.IngestionLiveView.FinalizeForm do
       end
 
     ~L"""
-    <div id="finalize_form" class="finalize-form finalize-form--<%= @visibility %>">
```

Comment:
does the other steps in ingestion need the same touch or do they work the same?

Reply:
We just need it for the first and last steps. This is so the line that extends between the step numbers doesn't overflow